app/main.py

```diff
@@ -2,24 +2,62 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: object, name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, obj: object, objtype: object) -> int:
+        return getattr(obj, self.protected_name)
+
+    def __set__(self, obj: object, value: int) -> None:
+        if not (self.min_amount <= value <= self.max_amount):
+            raise ValueError
```

Comment:
what's about other way?

HINT:
use `setattr`

Reply:
Need to add 
setattr(obj, self.protected_name, value) 
?